contrib_bots/bot_lib.py

```diff
@@ -81,8 +82,18 @@ def handle_message(message):
         logging.info('waiting for next message')
         if message_handler.triage_message(message=message,
                                           client=restricted_client):
+            query = None
+            # Handle the message that called through mention.
+            if message['is_mentioned']:
```

Comment:
Hmmmm, this is probably fine for now, but long term I think we should allow bots to have a little more freedom about how they handle at-mentions, even if they are in the middle or at the end of a message, and our library should  just provide hints.